phase1-coordinator/src/coordinator_state.rs

```diff
@@ -2118,15 +2370,49 @@ impl CoordinatorState {
 
                 warn!("Dropped {} from the ceremony", participant);
 
-                return Ok(DropParticipant::DropCurrent(DropCurrentParticpantData {
-                    participant: participant.clone(),
-                    bucket_id,
-                    locked_chunks,
-                    tasks,
-                    replacement: None,
-                }));
+                // Restart the round because there are no verifiers
+                // left for this round.
+                let reset_round = self.current_verifiers.is_empty() && self.finished_verifiers.is_empty();
```

Comment:
is it enough to have `self.current_verifiers.is_empty()`? Can you tell us more about the finished verifiers check here?

Reply:
The verifiers in the round get moved from the `current_verifiers` list to the `finished_verifiers` list when all their tasks are complete. If all the verifiers are considered complete for whatever reason, then the round will still be reset.

Reply:
at least that's how I was understanding it

Reply:
Is this case possible: one verifier finished, another one dropped?. This check will be produce therefore.

Reply:
Thanks for raising questions about this line, I think it deserves reconsideration.

Reply:
I referenced it in #305

Reply:
Yes it will produce false. Under the assumption that the remaining verifiers would be able to pick up the slack, but perhaps this isn't working yet and will be fixed in #305